README.md

```diff
@@ -5,7 +5,7 @@ Aby uruchomić aplikację kliencką należy:
  
 1. Mieć zainstalowanego npm'a. Można go zainstalować razem z node.js z tej strony: https://nodejs.org/en/
 2. Mieć zainstalowanego angulara. Można to zrobić przez wpisanie w terminalu: 
-npm install -g angular-cli
+npm install -g @angular-cli
```

Comment:
@angular/cli

Reply:
przy okazji można komendy dać zindentowane to wtedy się jako code blocki zrobią

Reply:
poprawiłem. Btw. to co było na początku też działa, ale faktycznie to z @ jest może ładniejsze

Reply:
Nvm. angular-cli to starsza wersja